lib/publiccloud/provider.pm

```diff
@@ -350,6 +350,8 @@ sub terraform_apply {
         my $sap_regcode = get_required_var('SCC_REGCODE_SLES4SAP');
         my $storage_account_name = get_var('STORAGE_ACCOUNT_NAME');
         my $storage_account_key = get_var('STORAGE_ACCOUNT_KEY');
+        # Enable specifying resource group name to allow running multiple tests simultaneously
+        my $resource_group = get_var('PUBLIC_CLOUD_RESOURCE_GROUP', 'qashapopenqa');
```

Comment:
Could you also add a description on `variables.md`?

Reply:
Sure, just updated the PR.